astroquery/mast/missions.py

```diff
@@ -33,7 +33,7 @@ class MastMissionsClass(MastQueryWithLogin):
     Class that allows direct programatic access to retrieve metadata via the MAST search API for a given mission.
     """
 
-    def __init__(self, *, mission='hst', service='search'):
```

Comment:
No I would have kept that in, that makes the arguments explicit keyword arguments. We've had things break previously when parameters were fed without explicitly stating which kwarg it was